melissa/profile_populator.py

```diff
@@ -3,7 +3,7 @@
 import subprocess
 import json
 from getpass import getpass
-import pywapi
+from melissa.utilities import pywapi
```

Comment:
Same here 
pywapi module test isn't required as here we are using the pywapi clone to add the functionality and because of this reason, the test for `melissa/profile_populator.py` at `melissa/tests/test_profile_populator.py` will give the result that if the `melissa/tests/test_profile_populator.py` test PASSES then its obvious pywapi module is working.